deploy_js_lambda.sh

```diff
@@ -0,0 +1,32 @@
+#/bin/bash
+
+STAGE=$1
+BUCKET=$2
```

Comment:
This works too - https://github.com/guardian/grid/blob/aa-dev-usage-lambda/cloud-formation/scripts/get-or-create-artifact-bucket.sh (part of https://github.com/guardian/grid/pull/1470).